source/administration/services/hdbpp/hdbpp-cm-interface.rst

```diff
@@ -0,0 +1,152 @@
+.. _hdbpp_cm-interface:
+
+:audience:`administrators, developers`, :lang:`C++`
+
+
+Configuration Manager interface
+-------------------------------
+
+.. contents::
+   :depth: 2
+
+.. Configuration Manager interface
+
+More in detail the device server exposes the following interface.
+
+Commands
+~~~~~~~~
+
+The commands availabile in the are summarized in commands-table.
+
++------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ArchiverAdd            | add a new instance to the archivers list; the instance must have been already created and configured via jive/astor and the device shall be running; as per release adding an device to an existing instance is not supported   |
++------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ArchiverRemove         | remove an from the list; neither the device instance nor the attributes configured are removed from the database                                                                                                                |
```

Comment:
"remove an EventSubscriver from..."
This is missing from almost all descriptions (was a latex \def in the original format...)

Reply:
thank you, especially for pointing out all the places it was missing !